missing information about API authorisation usages (#4284)
centreon/centreon-documentation: versioned_docs/version-24.10/monitoring/basic-objects/contacts-create.md

* The **Authentication Source** field specifies if the connection information comes from an LDAP directory or information

>   stored locally on the server.

Author: ```suggestion
* The **Authentication Source** field specifies if the connection information comes from an LDAP directory or information stored locally on the server.
```

> * The next 3 fields allow users to perform [API calls](../../api/rest-api-v1.md#api-calls) (note: our API documentation is written for developers familiar with HTTP requests and JSON).

Author: ```suggestion
* The next 3 fields are for authorizing users to perform [API calls](../../api/rest-api-v1.md#api-calls) (note: our API documentation is written for developers familiar with HTTP requests and JSON).
```

> * The next 3 fields are for authorizing users to perform [API calls](../../api/rest-api-v1.md#api-calls) (note: our API documentation is written for developers familiar with HTTP requests and JSON).

Author: ```suggestion
* The next 3 fields are for authorizing users to perform calls to our [API v1](../../api/rest-api-v1.md#api-calls) and [API v2](https://docs-api.centreon.com/api/centreon-web/24.10/) (note: our API documentation is written for developers familiar with HTTP requests and JSON).
```

> * The next 3 fields are to allow the user to perform [API calls](../../api/rest-api-v1.md#api-calls) (note: our API calls documentation is written for developers familiar with HTTP requests and JSON).

Reviewer: ```suggestion
* The next 3 fields allow users to perform [API calls](../../api/rest-api-v1.md#api-calls) (note: our API documentation is written for developers familiar with HTTP requests and JSON).
```

> * The next 3 fields are to allow the user to perform [API calls](../../api/rest-api-v1.md#api-calls) (note: our API calls documentation is written for developers familiar with HTTP requests and JSON).

Reviewer: là tu es dans le contexte du topic global sur la création de ocntacts, il faut donc traiter et le cas de l'API v1 et celui de l'API v2

> - The **Configuration API** can only be called by administrators, even if the corresponding field is checked for a non-admnistrator user.

Reviewer: ```suggestion
- The **Configuration API** can only be called by administrators, even if the corresponding field is checked for a non-administrator user.
```

> - The [**Realtime API**](../../api/rest-api-v1.md#realtime-information) can be called by non-administrator as long as the corresponding field is checked.

Reviewer 2: ```suggestion
- The [**Realtime API**](../../api/rest-api-v1.md#realtime-information) can be called by a non-administrator user as long as the corresponding field is checked.
```

> - Administrators are able to call both the **Configuration API** and the [**Realtime API**](../../api/rest-api-v1.md#realtime-information) regardless of the corresponding field of the api being checked.

Reviewer 2: ```suggestion
- Administrators are able to call both the **Configuration API** and the [**Realtime API**](../../api/rest-api-v1.md#realtime-information) regardless of the corresponding field of the API being checked.
```

> - Administrators are able to call both the **Configuration API** and the [**Realtime API**](../../api/rest-api-v1.md#realtime-information) regardless of the corresponding field being checked in both. This is true for both v1 and v2.

Author: ```suggestion
- Administrators are able to call both the **Configuration API** and the [**Realtime API**](../../api/rest-api-v1.md#realtime-information) regardless of the corresponding field being checked in both. This is true for both v1 and v2. They are also the only ones allowed to use [**CLAPI**](clapi.md) while others can only use Rest API.
```

> - Administrators are able to call both the **Configuration API** and the [**Realtime API**](../../api/rest-api-v1.md#realtime-information) regardless of the corresponding field being checked in both. This is true for both v1 and v2. They are also the only ones allowed to use [**CLAPI**](clapi.md) while others can only use Rest API.

Author: ```suggestion
- Administrators are able to call both the **Configuration API** and the [**Realtime API**](../../api/rest-api-v1.md#realtime-information) regardless of the corresponding field being checked in both. This is true for both v1 and v2. They are also the only ones allowed to use [**CLAPI**](../../api/clapi.md) while others can only use Rest API.
```

* The **Authentication Source** field specifies if the connection information comes from an LDAP directory or information
  stored locally on the server.

> * To perform [API calls](../../api/rest-api-v1.md#configuration)

Reviewer: garder la présentation type référence (=commencer par le nom des champs)

* The **Authentication Source** field specifies if the connection information comes from an LDAP directory or information
  stored locally on the server.
*To perform [API calls](../../api/rest-api-v1.md#api-calls)
- The **Configuration API** can only be called by administrators, even if the corresponding field is checked for a non-admnistrator user.
- The [**Realtime API**](../../api/rest-api-v1.md#realtime-information) can be called by non-administrator as long as the corresponding field is checked.

> - Administrators are able to call both the **Configuration API** and the [**Realtime API**](../../api/rest-api-v1.md#realtime-information) regardless of the corresponding field of the api being checked

Author: ```suggestion
- Administrators are able to call both the **Configuration API** and the [**Realtime API**](../../api/rest-api-v1.md#realtime-information) regardless of the corresponding field of the api being checked.
```

> 
> 
> 

Author: ```suggestion
```

* The **Authentication Source** field specifies if the connection information comes from an LDAP directory or information
  stored locally on the server.

> *To perform [API calls](#api-calls)

Author: ```suggestion
*To perform [API calls](../api/rest-api-v1.md#api-calls)
```

- The **Configuration API** can only be called by administrators.

> - The [**Realtime API**](#realtime-information) can be called by non-administrator as long as the corresponding field is checked.
> - Administrators are able to call both the **Configuration API** and the [**Realtime API**](#realtime-information) regardless of if the individual field of the api is checked

Author: ```suggestion
- The [**Realtime API**](../api/rest-api-v1.md#realtime-information) can be called by non-administrator as long as the corresponding field is checked.
- Administrators are able to call both the **Configuration API** and the [**Realtime API**](../api/rest-api-v1.md#realtime-information) regardless of if the individual field of the api is checked
```

> In all three instances, the corresponding fields can be found in the [**Centreon Authentication**](../monitoring/basic-objects/contacts-create.md#centreon-authentication) tab when configuring a user.

Author: ```suggestion
In all three instances, the corresponding fields can be found in the [**Centreon Authentication**](#centreon-authentication) tab when configuring a user.
```
